src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -82,6 +84,7 @@ export class ListingCreateComponent implements OnInit {
                 this.initiateListing();
             }
             this.initiateForm();
+            this.initiateNotes();
```

Comment:
hym, within my story I was changing a few bits of this component and I'm thinking that we should refactor notes component a bit. 
You might ask why? Right now create listing component knows too match about notes. As it initializes notes, sends them to API. 
Right now our tree looks like this
Container
  |-ListingCreateComponent
    |-NotesComponent

(this is kind against out approach Container and it's components
@radoslawlandowskikainos  I would say that it should look like this:
ParentContainer (here should be the store, and api releted stuff, and its components should be dummy)
  |-ListingComponenet 
  |-NotesComponent

This would give us a few advantages, smaller, more readable components == more reusable components, we will continue the container component pattern

WDUT? I'm open for discussion

Reply:
Yeah, I agree, there should be one more layer above ListingCreate component to make it dummier and unaware of the State. It would also be beneficial to use some inheritance, as Szymon once suggested, to distinguish between edit and create mode in ListingCreate component. Your comments are 100% valid. In my next story I'm modifying the ListingCreateComponent so I will think and try to refactor it as you propose. We'll probably talk that through on monday.